x-pack/test/api_integration/apis/ml/system/info.ts

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { getCommonRequestHeader } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  const expectedDefaults = {
+    anomaly_detectors: {
+      model_memory_limit: '1gb',
+      categorization_examples_limit: 4,
+      model_snapshot_retention_days: 10,
+      daily_model_snapshot_retention_after_days: 1,
+      categorization_analyzer: {
+        char_filter: ['first_line_with_letters'],
+        tokenizer: 'ml_standard',
+        filter: [
+          {
+            type: 'stop',
+            stopwords: [
+              'Monday',
+              'Tuesday',
+              'Wednesday',
+              'Thursday',
+              'Friday',
+              'Saturday',
+              'Sunday',
+              'Mon',
+              'Tue',
+              'Wed',
+              'Thu',
+              'Fri',
+              'Sat',
+              'Sun',
+              'January',
+              'February',
+              'March',
+              'April',
+              'May',
+              'June',
+              'July',
+              'August',
+              'September',
+              'October',
+              'November',
+              'December',
+              'Jan',
+              'Feb',
+              'Mar',
+              'Apr',
+              'May',
+              'Jun',
+              'Jul',
+              'Aug',
+              'Sep',
+              'Oct',
+              'Nov',
+              'Dec',
+              'GMT',
+              'UTC',
+            ],
+          },
+          {
+            type: 'limit',
+            max_token_count: '100',
+          },
+        ],
+      },
+    },
+    datafeeds: {
+      scroll_size: 1000,
+    },
+  };
+
+  async function runRequest(user: USER) {
+    const { body, status } = await supertest
+      .get(`/internal/ml/info`)
+      .auth(user, ml.securityCommon.getPasswordForUser(user))
+      .set(getCommonRequestHeader('1'));
+    ml.api.assertResponseStatusCode(200, status, body);
+
+    return body;
+  }
+
+  describe('POST ml/info', function () {
+    describe('get ml info', () => {
```

Comment:
Do we actually need a nested describe? And it seems like it should be `GET` instead 
```suggestion
  describe('GET ml/info', function () {
```

Reply:
Updated in https://github.com/elastic/kibana/pull/187474/commits/9596decdd43531fe8661ddfee84d527a8d885c02